lib/s3Helper.js

```diff
@@ -93,6 +93,15 @@ function compareS3Key (a, b) {
   }
 }
 
+module.exports.extractPseudoKey = function (a) {
+  const otherParts = a.Key.substring(47).split('/')
```

Comment:
can we use `parseS3Key` here instead of duplicating parsing logic?

Reply:
Yes, sure. Fixed this.